doc/website/blog/serverless-graphql-using-aws.md

```diff
@@ -0,0 +1,599 @@
+You should now see a form where you may insert the API name.
+For the present tutorial, we type "Todo", see the screenshot below, and click the "Create" button.
+
+<div style={{textAlign: 'center'}}>
+  <img alt="Screenshot of creating a new AWS AppSync API resource" src="https://entgo.io/images/assets/appsync/create-resources.png" />
+  <p style={{fontSize: 12}}>Creating a new API resource in AWS AppSync</p>
+</div>
+
+After creating the AppSync API, you should see a landing page showing a panel to define the schema, a panel to query the API, and a panel on integrating AppSync into your app as captured in the screenshot below.
+
+<div style={{textAlign: 'center'}}>
+  <img alt="Screenshot of the landing page of the AWS AppSync API" src="https://entgo.io/images/assets/appsync/getting-started.png" />
+  <p style={{fontSize: 12}}>Landing page of the AWS AppSync API</p>
+</div>
+
+Click the "Edit Schema" button in the first panel and replace the previous schema with the following GraphQL schema:
+
+```graphql
+input AddTodoInput {
+	title: String!
+}
+
+type AddTodoOutput {
+	todo: Todo!
+}
+
+type Mutation {
+	addTodo(input: AddTodoInput!): AddTodoOutput!
+	removeTodo(input: RemoveTodoInput!): RemoveTodoOutput!
+}
+
+type Query {
+	todos: [Todo!]!
+	todo(id: ID!): Todo
+}
+
+input RemoveTodoInput {
+	todoId: ID!
+}
+
+type RemoveTodoOutput {
+	todo: Todo!
+}
+
+type Todo {
+	id: ID!
+	title: String!
+}
+
+schema {
+	query: Query
+	mutation: Mutation
+}
+```
+
+After replacing the schema, a short validation runs and you should be able to click the "Save Schema" button on the top right corner and find yourself with the following view:
+
+<div style={{textAlign: 'center'}}>
+  <img alt="Screenshot AWS AppSync: Final GraphQL schema for AWS AppSync API" src="https://entgo.io/images/assets/appsync/final-schema.png" />
+  <p style={{fontSize: 12}}>Final GraphQL schema of AWS AppSync API</p>
+</div>
+
+If we sent GraphQL requests to our AppSync API, the API would return errors as no resolvers have been attached to the schema.
+We will configure the resolvers after deploying the Ent function via AWS Lambda.
+
+Explaining the present GraphQL schema in detail is beyond the scope of this tutorial.
+In short, the GraphQL schema implements a list todos operation via `Query.todos`, a single read todo operation via `Query.todo`, a create todo operation via `Mutation.createTodo`, and a delete operation via `Mutation.deleteTodo`.
+The GraphQL API is similar to a simple REST API design of an `/todos` resource, where we would use `GET /todos`, `GET /todos/:id`, `POST /todos`, and `DELETE /todos/:id`.
+For details on the GraphQL schema design, e.g., the arguments and returns from the `Query` and `Mutation` objects, I follow the practices from the [GitHub GraphQL API](https://docs.github.com/en/graphql/reference/queries).
```

Comment:
You did an amazing job here by explaining it in a few sentences.

Reply:
Thanks!